src/components/cc-button/cc-button.stories.js

```diff
@@ -157,12 +158,25 @@ As you can see here, \`hide-text\` can only be used if there is an \`image\` or
 });
 
 export const accessibleName = makeStory(conf, {
+  css: `
+    div {
+      margin-top: 2em;
+      margin-bottom: 0.5em;
+    }
+  `,
   dom: (container) => {
     container.innerHTML = `
+        <cc-notice intent="warning"><span slot="message">The <code>accessible-name</code> attribute is deprecated in favor of <code>a11y-name</code></span></cc-notice>
         <p>The accessible name can be checked by using the accessibility inspector of your browser.</p>
         <p>You may also hover the buttons because we populate the <code>title</code> attribute with the same value.</p>
+        
```

Comment:
**suggestion:** this is debatable and it's only my opinion but since we're testing the deprecate process: I would remove documentation for deprecated stuff.

I would keep the notice (nice one by the way), but I would remove the whole example and explanation about `accessible-name`.

We keep `accessible-name` for people who are already using it but we don't want new people to use it. We should only promote the new way.